app/views/shared/bulk_updates/forms/_location_keywords.html.erb

```diff
@@ -0,0 +1,116 @@
+<h2>Location Keywords</h2>
+
+<fieldset>
+  <%= update_type_select('location_keywords') %>
+</fieldset>
+
+<fieldset class="bulk-updates-find is-hidden">
+  <h4 class="title">Find Location Keyword</h4>
+  <p class="form-description space-bot">Update all collections that have location keywords that match the criteria specified below.</p>
+
+  <div class="row">
+    <div class="col-6">
+      <%= label_tag 'location_category', 'Category' %>
+      <%= text_field_tag('find_value[LocationCategory]', nil, id: 'location_category', class: 'full-width location-keyword-find') %>
+    </div>
+    <div class="col-6">
+      <%= label_tag 'type', 'Type' %>
+      <%= text_field_tag('find_value[Type]', nil, id: 'type', class: 'full-width location-keyword-find') %>
+    </div>
+  </div>
+
+  <div class="row space-top">
+    <h5 class="space-bot">Subregions</h5>
+
+    <div class="col-3">
+      <%= label_tag 'subregion_1', 'Subregion 1' %>
+      <%= text_field_tag('find_value[Subregion1]', nil, id: 'subregion_1', class: 'full-width location-keyword-find') %>
+    </div>
+
+    <div class="col-3">
+      <%= label_tag 'subregion_2', 'Subregion 2' %>
+      <%= text_field_tag('find_value[Subregion2]', nil, id: 'subregion_2', class: 'full-width location-keyword-find') %>
+    </div>
+
+    <div class="col-3">
+      <%= label_tag 'subregion_3', 'Subregion 3' %>
+      <%= text_field_tag('find_value[Subregion3]', nil, id: 'subregion_3', class: 'full-width location-keyword-find') %>
+    </div>
+
+    <div class="col-3">
+      <%= label_tag 'detailed_location', 'Detailed Location' %>
+      <%= text_field_tag('find_value[DetailedLocation]', nil, id: 'detailed_location', class: 'full-width location-keyword-find') %>
+    </div>
+  </div>
+</fieldset>
+
+<fieldset class="bulk-updates-value is-hidden">
+  <h4 class="title">New Location Keyword</h4>
+  <p class="form-description space-bot">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
```

Comment:
We should just empty this div (I think you already emptied the Science Keywords version of this)